### ISSUE TITLE
Cannot read property 'options' of undefined in RequestHandlers/Sequential.js

### ISSUE THREAD

**Issue by Yuudaari** [hydrabolt/discord.js#2189]
**Please describe the problem you are having in as much detail as possible:**
I'm having a rare issue with my discord bot that crashes it after a large amount of time has passed. Here's the error:
```
TypeError: Cannot read property 'options' of undefined
    at item.request.gen.end (C:\Users\Mackenzie\GitHub\node\WARD\node_modules\discord.js\src\client\rest\RequestHandlers\Sequential.js:69:75)
    at then (C:\Users\Mackenzie\GitHub\node\WARD\node_modules\snekfetch\src\index.js:204:21)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```
Honestly, I have no idea what could even be causing this. I looked through the source and couldn't really figure anything out.

**Include a reproducible code sample here, if possible:**
https://github.com/WaywardGame/WARD
Not sure where in here could be causing the issue, if it is caused by my code.

**Further details:**

- discord.js version: 11.2.1
- node.js version: 8.9.1
- Operating system: Windows 10
- Priority this issue should have – please be realistic and elaborate if possible: Annoying

- [ ] I found this issue while running code on a __user account__
- [ ] I have also tested the issue on latest master, commit hash:

Let me know if this could be something in my code causing this, and if you have any ideas of where to look. I can also help debug this if you have any ideas.

**Comment by gmedeadlystrike** [comment]
I have seen this issue on one of my bots that runs on the stable/npmjs build.

This bot simply listens for the ready event and then force caches a few hard-coded messages and updates them every so often via edits. 

Hopefully my bots slim usage helps narrow down this issue

**Comment by SpaceEEC** [comment]
This issue has been fixed in both the stable and 11.3-dev github branches, you can simply install one of them and you are good.
(stable for just this bugfix, 11.3-dev also has new features and probably other bugfixes)

Relevant commit: https://github.com/hydrabolt/discord.js/commit/69b7d5d58e7638f42476bb3a5dd5eb15f5f4a90a

**Comment by umithyo** [comment]
I am still getting this error on the latest version.
